flink-python/src/main/java/org/apache/flink/table/runtime/operators/python/aggregate/PythonStreamGroupTableAggregateOperator.java

```diff
@@ -39,30 +37,19 @@
 	@VisibleForTesting
 	protected static final String STREAM_GROUP_TABLE_AGGREGATE_URN = "flink:transform:stream_group_table_aggregate:v1";
 
-	private final PythonAggregateFunctionInfo aggregateFunction;
-
-	private final DataViewUtils.DataViewSpec[] dataViewSpecs;
-
 	public PythonStreamGroupTableAggregateOperator(
 		Configuration config,
 		RowType inputType,
 		RowType outputType,
-		PythonAggregateFunctionInfo aggregateFunction,
```

Comment:
What's the purpose of this change?

Reply:
When there are multiple flat_maps, an additional PythonAggregateFunctionInfo of count(*) will be added to handle the situation of retract.